### ISSUE TITLE
mod_filestore does too many checks

### ISSUE THREAD

**Issue by CyBeRoni** [zotonic/zotonic#1231]
mod_filestore does a bunch of checks to verify settings. I have permissions set up so it can't delete files. This causes checks to fail and me to be unable to use the module until I adjust permissions.

It would be nice to be able to tell the module 'I know what I'm doing just do what I tell you to do'.

**Comment by mworrell** [comment]
So, basically, you want to be able to set a filestore as non-deletable?

**Comment by CyBeRoni** [comment]
In this case, yes (ties in with my earlier request about making S3-storage more useful for multi-instance use too) but the issue is about checks that incorrectly prevent usage. 

I think it's fine to have a 'check settings' button that does a bunch of checks so you can be sure you have the settings correct (it would then be verbose about the success of the various operations it performs) but having the checks be successful I don't think should be a prerequisite for use of the module. Case in point: it works fine without the ability to delete files.

TL;DR: there are too many checks for too many things.

**Comment by ddeboer** [comment]
We’re solving the read-only issue in #1488.

@CyBeRoni Can you elaborate on which of the other checks fail against with your preferred permission settings?
